dask/dataframe/io/parquet/core.py

```diff
@@ -217,7 +217,11 @@ def read_parquet(
             else:
                 # Multiple sorted columns found, cannot autodetect the index
                 warnings.warn(
-                    "Multiple sorted columns found, cannot autodetect index",
+                    "Multiple sorted columns found %s, cannot\n "
+                    "autodetect index. Will continue without an index.\n"
+                    "To pick an index column, use the index= kwargs; to \n"
```

Comment:
Change "index=kwargs" to `index keyword argument`? Or at least make clear that it's a single argument, right?